### DISCUSSION TITLE
Change cursor on button hover

### DISCUSSION THREAD

**Discussion by miftahulrespati** [filamentphp/filament#2949] [Help]
Is there any method of `ButtonAction` that can change the mouse cursor when hovering over the button?

I'm creating a button with an authorization that only several roles can click the button. But instead of hiding the button, I want to disable the button with the **default cursor** instead of the **pointer cursor** (👆).

**Comment by themahdavi** [comment]
Hi 
if you create your own button can use @class directive and check if authorize show cursor-pointer else cursor-not-allowed !

if use filament button component you need send pr for this functionality

**Reply by studiowizjo** [reply]
Just use `disabled()` method on it =).
